class AuthException(Exception):
    pass


class MaxRetriesError(Exception):
    pass
